test-cases/longevity/longevity-large-partition-4days.yaml

```diff
@@ -13,6 +19,8 @@ instance_type_db: 'i3en.3xlarge'
 
 nemesis_class_name: 'SisyphusMonkey'
 nemesis_seed: '014'
+instance_type_loader: 'c5n.9xlarge'
```

Comment:
This one should be tested with a cheaper loader, we saw that the loader isn't that stressed.

Reply:
c5n.4xlarge should be good enough or even more than enough.

Reply:
Addressed in opened https://github.com/scylladb/scylla-cluster-tests/pull/4244